CONTRIBUTING.md

```diff
@@ -556,6 +556,7 @@ Pull requests should not be squash merged in order to keep complete commit messa
 If you removed packages or made some major NixOS changes, write about it in the release notes for the next stable release in [`nixos/doc/manual/release-notes`](./nixos/doc/manual/release-notes).
 
 ### File naming and organisation
+[file-naming-and-organisation]: #file-naming-and-organisation
```

Comment:
Unfortunately these only work locally in the file, changing the title will still break pages linking to it..

What _should_ work, and I really think we should have that, is custom HTML anchors like

```markdown
<a name="location"></a>
```

Can you try that and confirm whether it works?

Reply:
Is pandoc commonmark notation supported?
```markdown
### File naming and organisation {#location}
```

Reply:
Unfortunately not